components/idp-mgt/org.wso2.carbon.idp.mgt/src/test/java/org/wso2/carbon/idp/mgt/util/IdPManagementUtilTest.java

```diff
@@ -0,0 +1,241 @@
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.idp.mgt.util;
+
+
+import org.mockito.Mock;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.testng.PowerMockTestCase;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import org.wso2.carbon.identity.application.common.model.IdentityProvider;
+import org.wso2.carbon.identity.application.common.model.IdentityProviderProperty;
+import org.wso2.carbon.identity.application.common.util.IdentityApplicationConstants;
+import org.wso2.carbon.identity.application.common.util.IdentityApplicationManagementUtil;
+import org.wso2.carbon.identity.common.testng.WithCarbonHome;
+import org.wso2.carbon.identity.core.util.IdentityUtil;
+import org.wso2.carbon.idp.mgt.IdentityProviderManagementClientException;
+import org.wso2.carbon.idp.mgt.IdentityProviderManagementServerException;
+import org.wso2.carbon.idp.mgt.IdentityProviderManager;
+import org.wso2.carbon.idp.mgt.internal.IdPManagementServiceComponent;
+import org.wso2.carbon.idp.mgt.util.IdPManagementConstants.ErrorMessage;
+import org.wso2.carbon.user.core.service.RealmService;
+import org.wso2.carbon.user.core.tenant.TenantManager;
+
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.when;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.testng.Assert.assertEquals;
+
+import static org.wso2.carbon.identity.application.common.util.IdentityApplicationConstants.REMEMBER_ME_TIME_OUT;
+import static org.wso2.carbon.identity.application.common.util.IdentityApplicationConstants.REMEMBER_ME_TIME_OUT_DEFAULT;
+import static org.wso2.carbon.identity.application.common.util.IdentityApplicationConstants.SESSION_IDLE_TIME_OUT;
+import static org.wso2.carbon.identity.application.common.util.IdentityApplicationConstants.SESSION_IDLE_TIME_OUT_DEFAULT;
+
+
+/**
+ * Unit test cases for IdPManagementUtil.
+ */
+@WithCarbonHome
+@PrepareForTest({IdentityUtil.class , IdentityApplicationConstants.class, IdentityProviderManager.class,
+        IdentityApplicationManagementUtil.class, IdPManagementServiceComponent.class})
+public class IdPManagementUtilTest extends PowerMockTestCase {
+
+    @Mock
+    private IdentityProviderManager mockedIdentityProviderManager;
+    @Mock
+    private IdentityProvider mockedIdentityProvider;
+    @Mock
+    private IdentityProviderProperty mockedIdentityProviderProperty;
+    @Mock
+    private TenantManager mockedTenantManager;
+    @Mock
+    private RealmService mockedRealmService;
+
+
+    @DataProvider
+    public Object[][] getTenantIdOfDomainData() {
+        return new Object[][]{
```

Comment:
Keep new line between method signature and implementation.

Reply:
Addressed with 651d11d51c0e8a7fb7bbc837c8a614d5b1ca007b